Makefile

```diff
@@ -20,59 +20,24 @@ cythonize: .install-cython $(PYXS:.pyx=.c)
 	pip install -r requirements/dev.txt
 	@touch .install-deps
 
-.PHONY: lint
-lint: isort-check black-check flake8 mypy
+.PHONY: pre-commit
+pre-commit:
+	pre-commit run --all-files
```

Comment:
Here's a recommended invocation example: https://github.com/ansible/pylibssh/blob/bd53dd8/tox.ini#L236. It's especially useful to have `--show-diff-on-failure` in CI, less so locally but it also comes in handy to review  what mutations have been applied.
Also, dunno how it's possible with make but in tox it's common to allow end-users to bypass their own args to pre-commit so while `tox -e lint` runs `pre-commit run --all-files`, `tox -e lint -- flake8 --files x/y/z` would run `pre-commit run flake8 --files x/y/z` which is useful when trying to go through what a specfic linter reports separately in a more responsive manner.

Reply:
Done, thanks!